### PR TITLE
add typescript typings

### PR DESCRIPTION
**Pull request by vitalics**
create typings from [docs](https://github.com/DevExpress/testcafe-browser-provider-electron/blob/master/README.md#helpers)

### CODE REVIEW

File: ts-defs/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import { Accelerator, BrowserWindow, NativeImage, MenuItem, Menu } from "electron";
+
+declare module 'testcafe-browser-provider-electron' {
+    type DefinedMenuItemsActions =
```
Comment (AndreyBelym) [inline ts-defs/index.d.ts]: Could you please explain what is the purpose of this enumeration? AFAIK Electron apps don't have any predefined menu items.

Comment (vitalics) [reply ts-defs/index.d.ts]: Common electron apps have this item and for better IDE supporting such as VSCode it would be the best option

Comment (AndreyBelym) [reply ts-defs/index.d.ts]: I've checked a similar example in VS Code, and looks like it ignores the string enum type and uses just string as a type name. Also, it doesn't suggest enum values as parameter values. 
##### x.ts
```typescript
type Enum = 'A' | 'B' | 'C';

export default function (arg: string [] | string | object | number | Enum) {
    
};
```

##### y.ts
```typescript
import x from './x';

x()
```

image

So I think this type is redudant here. You can always create a such type in your tests, or even create a new package that will contain common menu item names.

**Comment by AndreyBelym** [comment]
Thank you for your contribution, I've published `testcafe-browser-provider-electron@0.0.8` with the typings.

**Comment by gimdongwoo** [comment]
@AndreyBelym There is not include `ts-defs/index.d.ts` on published npm module `testcafe-browser-provider-electron@0.0.8`.

Screen Shot 2019-05-29 at 1 05 14 AM

**Comment by AndreyBelym** [comment]
You're right, I forgot to add them to the `files` list in `package.json`. I've fixed it with https://github.com/DevExpress/testcafe-browser-provider-electron/commit/84de0afddbfb5d14c0ca75f6a73cdff1b698810a and published version `0.0.9` with definitions included.

**Pull request merged**